src/main/python/apache/aurora/executor/bin/thermos_executor_main.py

```diff
@@ -183,6 +183,14 @@
     help='Maximum number of rotated stdout/stderr logs emitted by the thermos runner.')
 
 
+app.add_option(
+    '--disk-log-level',
+    dest='disk_log_level',
+    type=str,
+    default='INFO',
```

Comment:
Default it to DEBUG? and then it can be changed on runtime with `--disk-log-level=INFO` but keep the default for backwards compatibility